src/main/java/org/threeten/extra/scale/UtcInstant.java

```diff
@@ -121,12 +121,18 @@
      * The Modified Julian Day, from the epoch of 1858-11-17.
      */
     private final long mjDay;
+
```

Comment:
When raising PRs in opens source projects please adhere to the coding standards of that project, and avoid IntelliJ auto-refactoring. This project has no blank lines between fields, and a blank line after the last method.

Reply:
Resolved